### DISCUSSION TITLE
QOL issues

### DISCUSSION THREAD

**Discussion by polzhang** [mljar/mljar-supervised#786] [Q&A]
I love what you can do with MLJar, but I’ve noticed a few (very minor!) quality-of-life issues:

- I find it strange that you can't zoom in (increase the UI font size) using a keyboard shortcut. Zooming out works fine, though. (Using Ctrl -) The best I can manage is to set the zoom level back to default using Ctrl 0.
- Jupyter automatically adds an extra cell below the output after code execution, which feels quite disruptive (not sure if intentional)

Am I missing something painfully obvious? Does anyone know how I could address these small annoyances?

**Comment by pplonski** [comment]
Hi @polzhang!

Thank you for your questions. Are you using MLJAR Studio? You should be able to zoom in with `Ctrl` `Shift` `+`

Extra cell is added after cell execution, to prepare notebook for the next code.

What is your use case for MLJAR?
